e2e/benchmarks/local-benchmark/index.html

```diff
@@ -561,32 +636,62 @@ <h2>TensorFlow.js Model Benchmark</h2>
 
       const modelFolder = gui.addFolder('Benchmark');
       let modelUrlController = null;
-      let modelParameterFolder = null;
-      let inputSizeController = null;
-      modelFolder.add(state, 'benchmark', Object.keys(benchmarks)).name('models').onChange(async model => {
+      modelController = modelFolder.add(state, 'benchmark', Object.keys(benchmarks));
+      modelController.name('models').onChange(async model => {
         await showMsg(null);
+        const benchmark = benchmarks[state.benchmark];
         state.isModelChanged = true;
         state.isModelLoaded = false;
         if (inputSizeController !== null) {
           modelParameterFolder.remove(inputSizeController);
           inputSizeController = null;
         }
-        const benchmark = benchmarks[state.benchmark];
-        if (isInputSizeSupported()) {
-          inputSizeController = modelParameterFolder.add(state, 'inputSize', benchmark['supportedInput']).name('inputSizes').onChange(async inputSize => {
+        if (modelArchitectureController !== null) {
+          modelParameterFolder.remove(modelArchitectureController);
+          modelArchitectureController = null;
+        }
+
+        if (isParameterDefined('inputSizes')) {
+          inputSizeController = modelParameterFolder.add(state, 'inputSize', benchmark['inputSizes']).name('inputSizes').onChange(async inputSize => {
             state.inputSize = inputSize;
-            state.isInputSizeChanged = true;
+            state.isModelChanged = true;
           });
           // Current use first value as default.
-          const defaultInputSize = benchmark['supportedInput'][0];
+          let defaultInputSize = 0;
```

Comment:
```
let defaultInputSize = benchmark['inputSizes'][0];
if (isURLParameterDefined('inputSize'))
  defaultInputSize = urlState.get('inputSize');

Reply:
I prefer the original version, it reads once for all case.

Reply:
Either is ok for me:) My previous opinion is base on code simplicity.